tensorboard/plugins/debugger/README.md

```diff
@@ -157,14 +160,16 @@ Within each card displaying a tensor value, the user can slice the tensor (via
 
 ![Input boxes for slicing tensors](images/slicing_input_boxes.png)
 
-For example, suppose a tensor has a shape of (500, 100), applying a slicing of [::5, :2] will slicing the tensor every 5 indices along the first dimension and take only the first two indices along the second dimension.
+For example, suppose a tensor has a shape of (500, 100), applying a slicing of `[::5, :2]` will slice the tensor every 5 indices along the first dimension and take only the first two indices along the second dimension.
 
 #### Slicing based on Time (Tensor Value History)
 
-For each tensor, the time axis (history of the tensor's execution) is treated as an 1D array. Numpy-style slicing can be applied to time. For example, the default slicing of `-1` selects for the most recent value. However, if the user changes that slicing parameter to `:`, the full history of the tensor will be shown (and the rank of the tensor being visualized is increased by 1).
+For each tensor, the time axis (history of the tensor's execution) is treated as an 1D array. Numpy-style slicing can be applied to time. For example, the default slicing of `-1` selects the most recent value. However, if the user changes that slicing parameter to `:`, the full history of the tensor will be shown (and the rank of the tensor being visualized is increased by 1).
 
 # Limitations
 
-Hitting Ctrl+C (issuing a SIGINT signal) might fail to terminate execution for a model that is instrumented with `TensorBoardDebugWrapperSession` or its corresponding hook. In those cases, the user must manually kill the processes.
-
-The debugger dashboard does not yet support multiple users debugging at once.
+* Hitting Ctrl+C (issuing a SIGINT signal) might fail to terminate execution for a model that is instrumented with
+  `TensorBoardDebugWrapperSession` or its corresponding hook. The same limitation may be present in the tensorboard
+  process as well. In those cases, the user must manually
+  [kill](https://www.linux.com/learn/intro-to-linux/2017/5/how-kill-process-command-line) the processes.
```

Comment:
FWIW, in bash, an easier way to do this is just to suspend the job (Ctrl+Z) and then bash will print the job number (typically "1") and you can do `kill %1` where 1 == job number to kill it by PID.

That said, why does TensorBoardDebugWrapperSession cause Ctrl+C to fail to kill tensorboard?  That was unexpected when I was testing it just now, and it'd be nice to avoid that limitation if possible.

Reply:
Yeah, @caisq, how easy would it be to avoid the hang? I think this has to do with how the logic runs in optimized mode, but I need to investigate further.

Ctrl+Z also nicely suspends the job for me, although I had to use `ps -a` to list processes and then use `kill -9 188425` to immediately kill job.

Reply:
I believe this has to do with gRPC handles SIGINT. Anyway, I'll file a bug to keep track of the resolution.